libraries/core-react/src/components/EdsProvider/EdsProvider.stories.tsx

```diff
@@ -0,0 +1,187 @@
+import { useRef, useState } from 'react'
+import styled from 'styled-components'
+import { Story } from '@storybook/react'
+import { menu, accessible } from '@equinor/eds-icons'
+import { useArgs } from '@storybook/client-api'
+import {
+  TopBar,
+  Menu,
+  Button,
+  Icon,
+  EdsProvider,
+  EdsProviderProps,
+} from '../..'
+
+const Wrapper = styled.div`
+  margin: 32px;
+  display: grid;
+  grid-gap: 32px;
+  grid-template-columns: repeat(4, fit-content(100%));
+`
+
+export const Default: Story<EdsProviderProps['density']> = (args) => {
+  const [_, setState] = useArgs()
```

Comment:
I assume this was to update the "props table"? Didn't seem to work on my machine as I had no controls for this prop in the props table as we usually have...